lib/streamlit/string_util.py

```diff
@@ -15,6 +15,7 @@
 import re
 import textwrap
 
+from typing import cast, Any
 from datetime import datetime
```

Comment:
nit: (seems like some of this code already existed before this change, but might as well fix it now)

These two imports should come after the `import textwrap` line and be sorted alphabetically

Additionally, there should be a newline before the `from streamlit.errors ...` import to separate it from the stdlib imports.